pytest_plt/tests/test_pytest.py

```diff
@@ -0,0 +1,61 @@
+"""The main test file that should be run on a regular basis.
+
+The goal of this file is to test pytest_plt internals and run all other
+test files with various invocations of pytest to ensure that all possible
+ways of using this plugin are tested.
+
+By default, if you call ``pytest`` while in this repository, only this
+file will be run due to configuration in ``setup.cfg``. However, other
+test files can be run manually by passing them to ``pytest``.
+"""
+
+import os
+
+from pytest_plt import Mock
+
+pytest_plugins = ["pytester"]
+
+
+def test_mock():
+    mock = Mock()
+    assert isinstance(mock["item"], Mock)
+    assert mock * "Hi" == 1.0
+    assert mock.__file__ == "/dev/null"
```

Comment:
Maybe make a note that this attribute and all the ones below are coming from `Mock.__getattr__`.